qemu/qmp.go

```diff
@@ -719,6 +719,10 @@ func QMPStart(ctx context.Context, socket string, cfg QMPConfig, disconnectedCh
 		}
 	}
 
+	if q.version.Major < 5 {
+		return nil, nil, fmt.Errorf("govmm requires qemu version 5.0 or later, this is qemu (%d.%d)", q.version.Major, q.version.Minor)
```

Comment:
this is a big step and I don't see any problem with this, would you mind updating the README?

Reply:
Ok, done.